pyaedt/maxwell.py

```diff
@@ -1920,17 +1920,24 @@ def export_element_based_harmonic_force(
         self.odesign.ExportElementBasedHarmonicForce(output_directory, setup, freq_option, f1, f2)
         return output_directory
 
-    @pyaedt_function_handler
-    def edit_external_circuit(self, netlist_file_path, schematic_design_name):
+    @pyaedt_function_handler()
+    def edit_external_circuit(self, netlist_file_path, schematic_design_name, parameters=None):
         """
-        Edit the external circuit for the winding.
+        Edit the external circuit for the winding and allow editing of the circuit parameters.
 
         Parameters
         ----------
         netlist_file_path : str
-            Circuit netlist file path.
+            Path to the circuit netlist file.
         schematic_design_name : str
             Name of the schematic design.
+        parameters : dict, optional
+            Name and value of the circuit parameters.
+            It must be provided as a dictionary, where the key is the parameter name
+            and value is the parameter value.
```

Comment:
```suggestion
            Parameters must be provided as a dictionary, where the key is the parameter name
            and the value is the parameter value.
```
Having "It" as the subject wasn't clear enough.